subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -796,10 +796,49 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
         /* Those are handled by NTVDM */
         case SCS_WOW_BINARY:
         {
-            DisplayMessage(L"Trying to load '%S'.\n"
-                           L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
-                           L"Press 'OK' to continue.",
-                           ProgramName);
+            static const PCSTR AppName = "\"%ProgramFiles%\\otvdm\\otvdmw.exe\" ";
+
+            STARTUPINFOA si;
+            PROCESS_INFORMATION pi;
+            CHAR ExpName[MAX_PATH];
+
+            ExpandEnvironmentStringsA(AppName, ExpName, ARRAYSIZE(ExpName) - 1);
+            strcat(ExpName, "\"");         // Add double-quote before ProgramName
```

Comment:
No offense, but can we please stop merging code that uses `strcat`? It's 2021.

Reply:
Was suggested at the very beginning of review: https://github.com/reactos/reactos/pull/4113#discussion_r752767833

Reply:
Thank you.
Doesn't look like I can comment on the old thread, but for future reference: consistency is not an argument here. The only reason most of our code base uses unsafe string functions is because it's old _and_ bad code. There's no reason to add more of it, and strsafe is accessible in all parts of our code (and if you find somewhere that it doesn't work, let me know, I'll fix it for you).

Reply:
@ThFabba, Thanks for your comments. I will try and see if I can create a PR for this soon and replace these with strsafe functions. If you say that they should work, then maybe it won't be too difficult.

Reply:
See PR#4124 https://github.com/reactos/reactos/pull/4124